README.md

```diff
@@ -24,3 +24,4 @@ $ docker run -it countingup/nswag help
 ## Changelog
 
  - 2021-11-06 -- Update to Alpine 3.14 base image
+ - 2022-01-17 -- Rebuild to update base image for security vulns
```

Comment:
I think we've been doing descending order

Reply:
fixed it